src/ConnectorPermissions.ts

```diff
@@ -0,0 +1,86 @@
+import * as nearAPI from "near-api-js";
+import {KeyPair} from "near-api-js";
+import { Contract } from "near-api-js";
+import {Environment, environmentToContractNameInfix} from "./Utils";
+const { keyStores } = nearAPI;
+const { connect } = nearAPI;
+import { PERMISSIONS_CONTRACT_PREFIX, CALIMERO_CONTRACT_SUFFIX } from "./Constants";
+
+export class ConnectorPermissions {
+
+  shardName: string;
+  permissionsContract: Contract;
+
+  private constructor(shardName: string, permissionsContract: nearAPI.Contract)
+  {
+    this.shardName = shardName;
+    this.permissionsContract = permissionsContract;
+  }
+  
+  static async init(shardName: string,
+    env: Environment,
+    accountId: string,
+    keyPair: KeyPair): Promise<ConnectorPermissions> {
+
+    const networkId = "testnet";
+    const myKeyStore = new keyStores.InMemoryKeyStore();
+    await myKeyStore.setKey(networkId,
+      accountId,
+      keyPair);
+
+    const connectionConfig = {
+      networkId: networkId,
+      keyStore: myKeyStore,
+      nodeUrl: "https://rpc.testnet.near.org",
+      walletUrl: "https://wallet.testnet.near.org",
+      helperUrl: "https://helper.testnet.near.org",
+      explorerUrl: "https://explorer.testnet.near.org",
+      headers: {},
+    };
```

Comment:
similarly I would not keep the hardcoded connectionConfig in each file, move it to a shared class

Reply:
Good point, I have extraced this into a shared file and added mainnet config